test/Test/Pos/Block/Logic/VarSpec.hs

```diff
@@ -59,13 +59,13 @@ verifyEmptyMainBlock :: BlockProperty ()
 verifyEmptyMainBlock = do
     -- unsafeHead is safe here, because we explicitly request to
     -- generate exactly 1 block
-    emptyBlock <- fst . unsafeHead . getOldestFirst <$> bpGenBlocks (Just 1)
+    emptyBlock <- fst . unsafeHead . getOldestFirst <$> bpGenBlocks (Just 1) False
     whenLeftM (lift $ verifyBlocksPrefix (one emptyBlock)) stopProperty
 
 verifyValidBlocks :: BlockProperty ()
 verifyValidBlocks = do
     bpGoToArbitraryState
-    blocks <- map fst . toList <$> bpGenBlocks Nothing
+    blocks <- map fst . toList <$> bpGenBlocks Nothing True
```

Comment:
Wrapping every bool value into newtype is a tradeoff i would better avoid. Having lots of constructors + extra unpacking -- better go to definition and read the comment.